webapp/src/Form/Type/ContestType.php

```diff
@@ -76,34 +76,34 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ],
             'help' => 'Disable this to stop recording balloons. Usually you can just leave this enabled.',
         ]);
-        $builder->add('processAwards', ChoiceType::class, [
+        $builder->add('processMedals', ChoiceType::class, [
             'expanded' => true,
             'choices' => [
                 'Yes' => true,
                 'No' => false,
             ],
-            'help' => 'Whether to process medal (gold, silver, bronze) awards for this contest.',
+            'help' => 'Whether to process medals (gold, silver, bronze) for this contest.',
```

Comment:
nit: I don't really like the word "process" here.

I would suggest "enable medals" or "award medals" or "display medals".

Reply:
I agree. I think display is even better, since I believe it is only used for the scoreboard display. I will check if that is indeed the case.
Which makes me wonder, if you disable this, shouldn't we also not expose it in the `awards` API endpoint?

Reply:
Yes, don't show medals if there are no medals

Reply:
The term "process" stems from process balloons because there it actually influences whether we keep track of it at all instead of whether we just display it.

Reply:
Thijs, is that a vote to keep the word process?

Reply:
It's just a piece of trivia

Reply:
> Yes, don't show medals if there are no medals

So then we call the field 'Enable medals' I'd say. Since it fully disables it

Reply:
OK it already disables the API, so renaming the field is all that is left 😄